resources/translations/fi.edn

```diff
@@ -256,8 +258,10 @@
          :event "Tapahtuma"
          :events "Tapahtumat"
          :failed "Epäonnistui"
+         :has-accepted-licenses "Olet hyväksynyt käyttöehdot."
          :licenses "Käyttöehdot"
          :maxlength "(enintään %1 merkkiä)"
+         :must-accept-licenses "Saadaksesi oikeudet hakemiisi resursseihin, sinun pitää hyväksyä seuraavat käyttöehdot."
```

Comment:
"hakemiisi" sounds like applicant but this is for members too? not insisting you fix this